.flake8

```diff
@@ -1,2 +1,3 @@
 [flake8]
```

Comment:
Just so that I remember, flake8 is a code formatter (like prettier for NodeJS) correct?

Reply:
No. Flake8 is a code form checker. It makes sure that you use spaces instead of tabs, don't import unnecessary code, and more. It doesn't actually fix it though.

Black (or 2black in this case) is the program that actually fixes most of these errors.

(And then Isort sorts all of the import statements to be grouped together by type. Flake8 and Isort are standards for Python packages that I code in.)

Reply:
Ahh okay gotcha, thank you :D.